### PR TITLE
[tests] Add test case for PME save and restore APIs

### PR DESCRIPTION
**Pull request by cuiyanx**
I add this test point to [tests/test-pme.js](https://github.com/01org/zephyr.js/blob/master/tests/test-pme.js), but I have a issue `[FATAL]: 10`. 
So, I create a new file to test this.

**TOTAL:** 1 of 1 passed

Signed-off-by: Cui Yan <yanx.cui@intel.com>

### CODE REVIEW

**Comment by jimmy-huang** [comment]
@cuiyanx please rebase against master

File: tests/test-pme-save-restore.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+var file = "neurons.txt";
+
+var workState = function() {
+    var state;
+    var category = pme.classify(classifys);
+
+    if (category === pme.NO_MATCH) {
+        category = "no match";
+
```
Comment (jimmy-huang) [inline tests/test-pme-save-restore.js]: remove newline

File: tests/test-pme-save-restore.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    return state;
+}
+
+var trainings = [
+    [70, 70, 70, 70, 70],
+    [50, 50, 50, 50, 50],
+    [30, 30, 30, 30, 30]
+];
+
+var classifys = [50, 52, 54, 52, 50];
```
Comment (jimmy-huang) [inline tests/test-pme-save-restore.js]: classifys is not a word, maybe just use classify, or classifications

File: tests/test-pme-save-restore.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+var neurons = pme.saveNeurons();
+var jsonOut = JSON.stringify(neurons);
+fs.writeFileSync(file, jsonOut);
+console.log("Saved " + neurons.length + " neurons to file '" + file +
+            "' with " + jsonOut.length + " bytes");
+
+var workOnNormal = workState();
+
+pme.forget();
+console.log("Cleare neurons");
```
Comment (jimmy-huang) [inline tests/test-pme-save-restore.js]: Cleared

File: tests/test-pme-save-restore.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+pme.restoreNeurons(neurons);
+console.log("Restoring " + neurons.length + " neurons from file '" + file +
+            "' with " + stats.size + " bytes");
+
+var workOnRestore = workState();
+
+assert(workOnNormal === true &&
+       workOnForget === false &&
+       workOnRestore === true,
+       "PMEObject: neurons can be saved and restored");
```
Comment (jimmy-huang) [inline tests/test-pme-save-restore.js]: Is  PMEObject the test name?  It sounds like a variable but it is not in the JS, so  maybe remove it?

Comment (cuiyanx) [inline tests/test-pme-save-restore.js]: Yes, those are test function save and restore of PME object. So I give this test name like in [/tests/test-pme.js](https://github.com/01org/zephyr.js/blob/master/tests/test-pme.js) and other test case.

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**